app/components/main-map.js

```diff
@@ -43,8 +43,16 @@ export default Ember.Component.extend({
       const height = el.height();
       const width = el.width();
 
+      const fullWidth = $('.navigation-area').width();
+      // width of content area on large screens is 5/12 of full
+      const contentWidth = (fullWidth / 12) * 5;
+      const offset = -((width - contentWidth) / 2);
+
       const padding = Math.min(height, width) / 2.5;
-      return { padding: selected && (type !== 'zoning-district') ? padding : 0 };
+      return {
+        padding: selected && (type !== 'zoning-district') ? padding : 0,
+        offset: [offset, 0],
+      };
```

Comment:
@chriswhong what should this computed be doing in terms of the view? Having trouble figuring out how to test / verify. Thanks!

Reply:
^^ I have some ideas on how to approach this differently

Reply:
This offsets the fitbounds, I am not sure how you would test the output, you would need to be able to read the centerpoint of the map vs the center of the highlighted polygon somehow.

Reply:
Ah, I just mean manual testing. Why does zoning-district get padding but not others? Thanks!